.github/workflows/ci.yaml

```diff
@@ -0,0 +1,44 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version:
+        - '2.5'
```

Comment:
why 2.5?  It's EOL since March 2021 I think.  It's not in the .travis.yml either.

Reply:
copy paste fail from the other gem...i'll fix